include/slang/binding/AssignmentExpressions.h

```diff
@@ -90,7 +94,8 @@ class ConversionExpression : public Expression {
                                   const BindContext& context);
 
     static ConstantValue convert(EvalContext& context, const Type& from, const Type& to,
-                                 SourceRange sourceRange, ConstantValue&& value);
+                                 SourceRange sourceRange, ConstantValue&& value,
+                                 ConvertCast cast = ConvertCast::Implicit);
```

Comment:
I wouldn't default this parameter. We should force callers to say what kind of conversion they are doing.

Reply:
Yes, I will make the change.